File: src/main/java/com/prog5/items/BusinessCard.java
package com.prog5.items;

public class BusinessCard {
    private String ownerName;

    private String companyName;

    public BusinessCard(String ownerName, String companyName) {
        this.ownerName = ownerName;
        this.companyName = companyName;
    }

    @Override
    public String toString() {
        return "\nBusinessCard:" +
                "\nownerName: " + ownerName +
                ", companyName: " + companyName;
    }
}

File: BusinessCard.cs
namespace Items
{
  public class BusinessCard
  {
    public string Owner { get; set; }
    public string CompanyName { get; set; }

    public BusinessCard(string owner, string companyName)
    {
        Owner = owner;
        CompanyName = companyName;
    }
  }
}

